Make sure this passes even when there is Cython installed.

--- setuptools-72.2.0/setuptools/command/build_ext.py.orig
+++ setuptools-72.2.0/setuptools/command/build_ext.py
@@ -18,7 +18,7 @@
 
 try:
     # Attempt to use Cython for building extensions, if available
-    from Cython.Distutils.build_ext import build_ext as _build_ext  # type: ignore[import-not-found] # Cython not installed on CI tests
+    from Cython.Distutils.build_ext import build_ext as _build_ext  # type: ignore # Cython not installed on CI tests
 
     # Additionally, assert that the compiler module will load
     # also. Ref #1229.
